dataikuapi/dss/project.py

```diff
@@ -839,11 +839,11 @@ def list_jupyter_notebooks(self, active=False, as_type="object"):
         :returns: The list of the notebooks. If "as_type" is "names", each one as a string, if "as_type" is "objects", each one as a :class:`dataikuapi.dss.notebook.DSSJupyterNotebook`
```

Comment:
@apichery  Hi the return explanation here `If "as_type" is "names"` is wrong, maybe you forgot to change it. I guess you can just say: 
`If "as_type" is "listitems", then each one as a dict.`